jest.config.js

```diff
@@ -189,12 +189,10 @@ module.exports = {
   },
 
   // An array of regexp pattern strings that are matched against all source file paths, matched files will skip transformation
-  // transformIgnorePatterns: [
-  //   "\\\\node_modules\\\\",
-  //   "\\.pnp\\.[^\\\\]+$"
-  // ],
-  transformIgnorePatterns: ['/node_modules/@nypublicradio(?!nypr-design-system-vue3)'],
```

Comment:
this was the main culprit! I had it set to "nypr-design-system-vue3" not "@nypublicradio/nypr-design-system-vue3" 🙄